selfservice/strategy/oidc/provider_gitlab.go

```diff
@@ -33,6 +34,20 @@ func NewProviderGitLab(
 	}
 }
 
+func (g *ProviderGitLab) oauth2() *oauth2.Config {
+	return &oauth2.Config{
+		ClientID:     g.config.ClientID,
+		ClientSecret: g.config.ClientSecret,
+		Endpoint:     gitlab.Endpoint,
```

Comment:
Gitlab can also have a custom URL if it is self hosted, therefore we have the default endpoint and issuer URL.

Reply:
you are right, lemme check for that and update